hatasmota/discovery.py

```diff
@@ -514,6 +534,8 @@ def get_entity(
         return TasmotaStatusSensor(config=config, mqtt_client=mqtt_client)
     if platform == "switch":
         return TasmotaRelay(config=config, mqtt_client=mqtt_client)
+    if platform == "camera":
+        return TasmotaCamera(config=config, mqtt_client=mqtt_client)
```

Comment:
Please move this before cover to keep the alphabetical sorting

Reply:
Done.